components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -47,4 +47,34 @@ class ActionSheetTest: XCTestCase {
     // Then
     XCTAssertEqual(actionSheet.title, "New title")
   }
+
+  func testAccessibilityIdentifiers() {
+    // Given
+    var testIdentifiers: [String] = []
+    let section = 0
+    let actionCount = 10
+    for i in 0..<actionCount {
+      testIdentifiers += ["Test \(i)"]
+    }
+    let actionSheet = MDCActionSheetController(title: nil)
+
+    // When
+    for i in 0..<actionCount {
+      let action = MDCActionSheetAction(title: "Title", image: nil, handler: nil)
+      action.accessibilityIdentifier = testIdentifiers[i]
+      actionSheet.addAction(action)
+    }
+
+    // Then
+    XCTAssertEqual(actionSheet.view.subviews.count, 2)
+    for views in actionSheet.view.subviews {
```

Comment:
Something like:

```swift
let tableView = self.view.subviews.first { $0 is UITableView } as? UITableView
```

Might let you test that the able view exists before you enumerate it.

Reply:
Or:

```swift
let tableView = self.view.subviews.flatMap { $0 as? UITableView }.first
```